.github/workflows/docker.yml

```diff
@@ -71,7 +73,7 @@ jobs:
           # generate Docker tags based on the following events/attributes
           tags: |
             latest
-            type=semver,pattern={{version}}
+            type=raw,value=$tag_name
```

Comment:
this line also needs to change to 
```
type=raw,value=${{ env.tag_name }}
```

Also, usually, all env vars are typically all caps with underscores: `TAG_NAME`

Reply:
Thank you @vraravam 😄 Pushed a new commit to solve this. Can you take a look, please?